### DISCUSSION TITLE
Darkreader extention stops working periodically in Firefox

### DISCUSSION THREAD

**Discussion by Asheneyes** [darkreader/darkreader#6944] [Q&A] [answered]
Occationally my darkreader extention stops working in firefox. 

I'll enable the extension, and have the bar looking like this: 

![image](https://user-images.githubusercontent.com/85908490/135142909-c97adeee-1230-4b66-8b4b-48c2a2ff9f63.png)

And then a day or so later the bar would look like this.

![image](https://user-images.githubusercontent.com/85908490/135142991-f162b4cd-6391-4932-bbf9-d9a8ca123dfc.png)

But the Firefox extension page still has it enabled. 

![image](https://user-images.githubusercontent.com/85908490/135143215-1203b7ad-ba68-4aee-bd5f-3470c8a901cf.png)

Only after I toggled the extension on and off in the extension page would it re-enable for another day or so. 

Anyone know why this is happening, or, more specifically, how to ensure it remains enabled?

**Comment by Gusted** [comment, marked as answer]
> Only after I toggled the extension on and off in the extension page would it re-enable for another day or so.
> 
> Anyone know why this is happening, or, more specifically, how to ensure it remains enabled?

That's a pretty weird issue, extension itself don't have any control over this kind of state so it could be a hidden program doing this or a bug in Firefox that's doing this.

**Reply by Asheneyes** [reply]
Seems the issue stopped after a new firefox update yesterday. Maybe they saw this? Maybe it's been reported elsewhere. Ahh well. Fixed!
